test/sdk/restapi/swagger/model_googlerpc_status.go

```diff
@@ -0,0 +1,30 @@
+// Copyright 2022 Google LLC All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// This code was autogenerated. Do not edit directly.
+/*
+ * sdk.proto
+ *
+ * No description provided (generated by Swagger Codegen https://github.com/swagger-api/swagger-codegen)
+ *
+ * API version: version not set
+ * Generated by: Swagger Codegen (https://github.com/swagger-api/swagger-codegen.git)
+ */
+package swagger
+
+type GooglerpcStatus struct {
```

Comment:
```suggestion
type GoogleRPCStatus struct {
```

or

```suggestion
type RPCStatus struct {
```

Reply:
This file was auto-generated by Swagger Codegen. So even if I want to edit the file, I can't.